src/Vendor/Laravel/Artisan/Report.php

```diff
@@ -39,7 +41,7 @@ public function __construct() {
      * @return mixed
      */
     public function fire() {
-        $this->table = $this->getHelperSet()->get('table');
+        $this->table = new Table($this->output);
 
         $list = [];
 

```

Comment:
Line 62 you dont need $this->getOutput

http://symfony.com/doc/current/components/console/helpers/table.html